server/e2e/sync_test.go

```diff
@@ -0,0 +1,86 @@
+//go:build e2e
+// +build e2e
+
+package main
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/mattermost/mattermost/server/public/model"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestStopPluginSendMessageAndReplyToMSTeamsLinkedChannelStartPlugin(t *testing.T) {
+	t.Skip("Not implemented yet")
```

Comment:
Is this still intended to be skipped?

Reply:
Yes, this is for whenever we have the sync mechanism done